classes/class-uagb-post-assets.php

```diff
@@ -826,7 +826,8 @@ public function get_block_css_and_js( $block ) {
 		if (
 			'enabled' === \UAGB_Admin_Helper::get_admin_settings_option( 'uag_enable_motion_effects_extension', 'enabled' ) &&
 			( strpos( $name, 'uagb' ) === 0 ) &&
-			( ! empty( $block['attrs']['UAGMFXMouse'] ) )
+			( isset( $block['attrs']['UAGMFXMouse'] ) && $block['attrs']['UAGMFXMouse'] ) ||
```

Comment:
```suggestion
			( ! empty( $block['attrs']['UAGMFXMouse'] ) || ! empty( $block['attrs']['UAGMFXScroll'] ) )
```
can we try this, any specific reason of reverting this logic?

Reply:
Done ✅ 
The above suggested logic works as intended